OpenStack-nova-accounting/extract_accounting.py

```diff
@@ -21,7 +21,7 @@
    password = parser.get('auth', 'password')
    instance = parser.get('cloud','instance')
 except:
-   print('Unable to read from config file')
+   raise RuntimeError('Unable to read from config file')
    sys.exit(1)
```

Comment:
Nit: these `sys.exit(1)` can be dropped as it's effectively dead code
But it's not worth holding up the PR for